kernel/base/src/main/java/com/twosigma/beakerx/NamespaceClient.java

```diff
@@ -31,6 +31,8 @@
 
 public class NamespaceClient implements BeakerXClient {
 
+  public static final String NAMESPACE_CLIENT = NamespaceClient.class.getSimpleName() + ".getBeaker()";
```

Comment:
Better to expose "getBeakerX" instead of Beaker

Reply:
done